src/hooks/useENSRegistrationStepHandler.tsx

```diff
@@ -90,7 +97,9 @@ export default function useENSRegistrationStepHandler(observer = true) {
       const timeDifference = isTestingHardhat ? now - msBlockTimestamp : 0;
       const commitTransactionConfirmedAt = msBlockTimestamp + timeDifference;
       const secs = differenceInSeconds(now, commitTransactionConfirmedAt);
-      setSecondsSinceCommitConfirmed(secs);
+      setSecondsSinceCommitConfirmed(
+        secondsSinceCommitConfirmed < 0 ? 0 : secs
```

Comment:
if this value is -1 means that the user was on the screen while the commit tx was confirmed, so let's take "now" (0) as the time when it was confirmed